doc/developer.md

```diff
@@ -45,40 +48,50 @@ We recommend this for development work ONLY.
     ```
 
-7. Copy the configuration files
-    ```
-    cp /var/www/html/conf/apel_rest_api.conf /etc/httpd/conf.d/apel_rest_api.conf
-    cp /var/www/html/conf/ssl.conf /etc/httpd/conf.d/ssl.conf
-    cp /var/www/html/conf/cloudloader.cfg /etc/apel/cloudloader.cfg
-    cp /var/www/html/conf/cloudsummariser.cfg /etc/apel/cloudsummariser.cfg
-    ```
-
-8. Copy the script files
-    ```
-    cp /var/www/html/scripts/cloudsummariser /etc/cron.d/cloudsummariser
-    cp /var/www/html/scripts/run_cloud_summariser.sh /usr/bin/run_cloud_summariser.sh
-    cp /var/www/html/scripts/apeldbloader-cloud /etc/init.d/apeldbloader-cloud
-    ```
-
-9. Create log, run and spool directories
+7. Create etc, log, run and spool directories
    ```
+   mkdir /etc/apel
    mkdir /var/log/cloud
    mkdir /var/run/cloud
    mkdir -p /var/spool/apel/cloud/
    chown apache -R /var/spool/apel/cloud/
    ```
 
-10. To allow successful GET requests, you will need to register your APEL REST instance with the Indigo DataCloud IAM and add IAM variables in `/var/www/html/apel_rest/settings.py`. You will also need to register a second service (the querying test service), and authorise it by adding it's ID to `ALLOWED_FOR_GET`
+8. Symlink the local config files into the `/etc` directory. Note: these commands will override any existing configurations in those locations.
+    ```
+    ln -sf /var/www/html/conf/apel_rest_api.conf /etc/httpd/conf.d/apel_rest_api.conf
+    ln -sf /var/www/html/conf/ssl.conf /etc/httpd/conf.d/ssl.conf
+    ```
+
+9. To allow successful GET requests, you will need to register your APEL REST instance with the Indigo DataCloud IAM and add IAM variables in `/var/www/html/apel_rest/settings.py`. You will also need to register a second service (the querying test service), and authorise it by adding it's ID to `ALLOWED_FOR_GET`
     ```
+    IAM_HOSTNAME_LIST=
     SERVER_IAM_ID=
     SERVER_IAM_SECRET=
     ALLOWED_FOR_GET=
     ```
 
-11. Run `python manage.py collectstatic`
-
-12. Start Apache with `service httpd start`
+10. Run `python /var/www/html/manage.py collectstatic`
 
-13. Start the loader with `service apeldbloader-cloud start`
+11. Start Apache with `service httpd start`
 
-14. Navigate a web browser to "https://<hostname>/api/v1/cloud/record/summary/"
+12. Navigate a web browser to `https://<hostname>/api/v1/cloud/record/summary`
+
+## Optional: Set up a Docker-ized version of the APEL Server
+
+Deploying an APEL Server along with the REST interface and database is not necessary, but is useful for allowing data to flow from input via the REST interface to summarised output via the REST interface. For the purposes of developing the APEL Rest interface we treat the APEL Server as a 'black box', so we recommend deploying it as a Docker image.
+
+You will need to install Docker and Docker Compose first, see the [README.md](../README.md#running-the-docker-image-on-centos-7-and-ubuntu-1604) for a link to the instructions.
```

Comment:
"for a link to the instructions" - the link provided isn't the instructions themselves?

Reply:
my logic was rather than duplicate an external link that could die, point to where the link can be found. That way if the link dies it only has to be changed once.

Reply:
Fair enough.